File: Data Structures & Algorithms/remove-element/submission-4.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k=0;
        for (auto x:nums)
            if (x!=val)
                k++;
        int j=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]!=val)
            {
                nums[j]=nums[i];
                j++;
            }
        }
        return k;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        countN={}
        for i in nums:
            countN[i]=countN.get(i,0)+1
        arr=[]
        for num,cnt in countN.items():
            arr.append([cnt,num])
        arr.sort()

        res=[]
        while len(res)<k:
            res.append(arr.pop()[1])
        return res

